### PR TITLE
codesmooth 1.0.0

### PR DESCRIPTION
**Pull request by artur-kot**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/c/codesmooth.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Codesmooth < Formula
+  desc "Command-line interface for CodeSmooth tasks"
+  homepage "https://codesmooth.dev"
+  url "https://storage.googleapis.com/codesmooth-resources/public/cli/codesmooth-1.0.0.tar.gz"
+  sha256 "087a41437237861f7b49b3d3ab0b64b37efdec52e3843cda253c23a53e6cb9b4"
+  license "Apache-2.0"
```
Comment (chenrui333) [inline Formula/c/codesmooth.rb:6]: where is the source code?

File: Formula/c/codesmooth.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class Codesmooth < Formula
+  desc "Command-line interface for CodeSmooth tasks"
+  homepage "https://codesmooth.dev"
+  url "https://storage.googleapis.com/codesmooth-resources/public/cli/codesmooth-1.0.0.tar.gz"
```
Comment (chenrui333) [inline Formula/c/codesmooth.rb:4]: if you just distribute the binary, it would be a good fit for cask.

File: Formula/c/codesmooth.rb (hunk 1 above)
Comment (ArturKot95) [reply Formula/c/codesmooth.rb:6]: https://github.com/CodeSmooth-dev/codesmooth-cli
or inside the .tar.gz

File: Formula/c/codesmooth.rb (hunk 2 above)
Comment (ArturKot95) [reply Formula/c/codesmooth.rb:4]: ok, moving it to cask repo

**Pull request closed without merging**